.eslintrc.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  extends: ["react-app", "plugin:jsx-a11y/recommended", "prettier"],
+  plugins: ["jsx-a11y"],
+};
```

Comment:
I saw the `env` section was removed (from [here](https://github.com/codefordallas/codefordallas.github.io/pull/57#discussion_r1225943765)) - were these no longer needed?

Reply:
No that would be defining it twice. Its

Reply:
if you think it's better to do it the other way (as is now in the code) then I'm good with keeping it like this